agent/nmap_agent.py

```diff
@@ -406,7 +407,7 @@ def _emit_fingerprints(
                                 "port": data.get("port"),
                                 "schema": data.get("service"),
                                 "library_name": data.get("product"),
-                                "library_version": None,
+                                "library_version": data.get("product_version"),
```

Comment:
do we want this to default to `""`, because if its not present that the value that this will have.

Reply:
No.